test/verify/check-shell-keys

```diff
@@ -21,9 +21,9 @@ import parent
 from testlib import *
 
 
-FP_MD5 = "93:40:9e:67:82:78:a8:99:89:39:d5:ba:e0:50:70:e1"
-FP_SHA256 = "SHA256:SRvBhCmkCEVnJ6ascVH0AoVEbS3nPbowZkNevJnXtgw"
-KEY = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDG4iipTovcMg0xn+089QLNKVGpP2Pgq2duxHgAXre2XgA3dZL+kooioGFwBQSEjbWssKy82hKIN/W82/lQtL6krf7JQWnT3LZwD5DPsvHFKhOLghbiFzSI0uEL4NFFcZOMo5tGLrM5LsZsaIkkv5QkAE0tHIyeYinK6dQ2d8ZsfmgqxHDUQUWnz1T75X9fWQsUugSWI+8xAe0cfa4qZRz/IC+K7DEB3x4Ot5pl8FBuydJj/gb+Lwo2Vs27/d87W/0KHCqOHNwaVC8RBb1WcmXRDDetLGH1A9m5x7Ip/KU/cyvWWxw8S4VKZkTIcrGUhFYJDnjtE3Axz+D7agtps41t test-name"
```

Comment:
I think I saw a test failure around that yesterday -- if this leads to non-trivial problems, I recommend to just break this up into a multi-line string for this PR. (It seems useful to test RSA keys somewhere, after all)

Reply:
Honestly, this is a place I'd rather use #noqa.

...or generate the key at run time...

Reply:
There are no fails related to this in the tests.  Just the curl thing...

Do you still prefer me to put back RSA?

Reply:
eh, I take it back?  What is this failing in *only* Firefox?  That needs to be looked into...

Reply:
I'm fine with moving to EC.

The [firefox failure](https://logs.cockpit-project.org/logs/pull-16944-20220209-055643-eee3a6c4-fedora-35-firefox/log.html#122) is "just" an unexpected message, looks like this test is missing `allow_restart_journal_messages()`? I suppose it does not actually *re*start, but that function enables the messages for "sudden" disconnects.

Reply:
Hilarious.  Trying to reproduce that locally gets me another fun fail: `testlib.Error: FAIL: Test completed, but found unexpected journal messages:
audit: type=1400 audit(1644413645.103:237): avc:  denied  { execute } for  pid=2126 comm="chpasswd" name="sss_cache" dev="vda5" ino=39569 scontext=unconfined_u:unconfined_r:passwd_t:s0-s0:c0.c1023 tcontext=system_u:object_r:sssd_exec_t:s0 tclass=file permissive=0`

Reply:
@allisonkarlitskaya : That is just our latest OS regression: https://github.com/cockpit-project/bots/issues/2898